src/main/java/uk/gov/justice/laa/crime/cfecrime/controllers/CfeCrimeController.java

```diff
@@ -27,14 +28,27 @@ public class CfeCrimeController {
             )
     )
     @PostMapping(produces = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<CfeCrimeResponse> invoke(@Parameter(
+    public ResponseEntity<Object> invoke(@Parameter(
             description = "CFE Crime Request",
             content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE,
                     schema = @Schema(implementation = CfeCrimeRequest.class)
             )
-    ) @Valid @RequestBody CfeCrimeRequest request) {
+    ) @Valid @RequestBody CfeCrimeRequest request)  {
 
-        CfeCrimeResponse response = RequestHandler.handleRequest(request);
+        CfeCrimeResponse response = null;
+        try {
+            response = RequestHandler.handleRequest(request);
```

Comment:
IMHO it would be nicer to have return ResponseEntity.ok(RequestHandler.handleRequest(request)) - WDYT?